ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/ProxyIntegrationTest.java

```diff
@@ -72,7 +88,7 @@ void postRequestToNonRootPathIsProxied() {
         response.ethSigner(RESPONSE_HEADERS, LOGIN_RESPONSE),
         "/login");
 
-    verifyEthNodeReceived(REQUEST_HEADERS, LOGIN_BODY, "/login");
+    verifyEthNodeReceived(REQUEST_HEADERS, LOGIN_BODY, Path.of("/", ROOT_PATH, "login").toString());
```

Comment:
Is using Path.of to calculate the expected path going to fail on a non-unix OS i.e. Windows? Perhaps just create a constant for the expected value as it is always the same?

Reply:
It should not fail on Windows as it is not used by underlying file system.

Reply:
The separator that is used to combine the strings using Path.of will be different on Windows as that is OS specific. i.e. think we will get `/\<ROOT_PATH>\login` instead of `/<ROOT_PATH>/login`

Reply:
yes, you are correct @jframe. https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...)

Reply:
This test still has use of Path.of looks like all the others got fixed though